spinalcordtoolbox/moco.py

```diff
@@ -865,3 +868,209 @@ def spline(folder_mat, nt, nz, verbose, index_b0 = [], graph=0):
 
     sct.printv('\n...Done. Patient motion has been smoothed', verbose)
     sct.printv('------------------------------------------------------------------------------\n', verbose)
+
+def moco_wrapper_interleaved(param):
+    """
+    Prepare even and odd datasets and run moco separately, then merge data back together
+    :param param: ParamMoco class
+    :return: None
+    """
+
+    orig_name = param.fname_data    # store orig name passed by user
+    file_data = 'data.nii'          # corresponds to the full input data (e.g. dmri or fmri)
+    file_mask = 'mask.nii'
+    file_moco_params_x = 'moco_params_x.nii.gz'
+    file_moco_params_y = 'moco_params_y.nii.gz'
+    file_moco_params_csv = 'moco_params.tsv'
```

Comment:
`file_moco_params_csv` should be renamed to `file_moco_params_tsv` otherwise it's confusing

Reply:
👍 Thanks, I renamed variable `file_moco_params_csv` to `file_moco_params` inside `moco_wrapper_interleaved`. However, there is also similar discrepancy in original `moco_wrapper`:

https://github.com/neuropoly/spinalcordtoolbox/blob/048c0a5c4703c48ad1aab52838bf4200f3666101/spinalcordtoolbox/moco.py#L132

I do not want modify code of original (working and tested) `moco_wrapper` inside this PR. I will fix it when this PR will be finished.

Reply:
> I do not want modify code of original moco_wrapper (which works and is tested) inside this PR. I will fix it when this PR will be finished (I opened separated issue for it #2792).

thanks, this is a sensible approach